api/src/pcapi/scripts/matching_acceslibre/main.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from math import ceil
+
+import click
+import sqlalchemy as sa
+
+from pcapi.core.offerers import api as offerers_api
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+from pcapi.utils.blueprint import Blueprint
+
+
+blueprint = Blueprint(__name__, __name__)
+logger = logging.getLogger(__name__)
+
+BATCH_SIZE = 1000
+
+
+@blueprint.cli.command("find_new_match_at_acceslibre")
+@click.option("--dry-run", type=bool, default=True)
+def find_new_match_at_acceslibre(dry_run: bool = False, start_from_batch: int = 1) -> None:
+    venues_query = (
+        offerers_models.Venue.query.outerjoin(offerers_models.Venue.accessibilityProvider)
+        .filter(
+            offerers_models.Venue.isPermanent == True,
+            offerers_models.Venue.isVirtual == False,
```

Comment:
Si cette fonction est finalement mergée, par homgénéité du code :
```suggestion
            offerers_models.Venue.isPermanent.is_(True),
            offerers_models.Venue.isVirtual.id_(False),
```